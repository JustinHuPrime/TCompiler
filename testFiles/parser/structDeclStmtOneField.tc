module foo;

void bar() {
  struct s {
    int x;
  };
}