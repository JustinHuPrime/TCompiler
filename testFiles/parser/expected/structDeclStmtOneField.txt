testFiles/parser/structDeclStmtOneField.tc (code):
FILE(1, 1, STAB(ENTRY(bar, FUNCTION(testFiles/parser/structDeclStmtOneField.tc, 3, 1, void()))), MODULE(1, 1, ID(1, 8, foo, REFERENCES())), FUNDEFN(3, 1, KEYWORDTYPE(3, 1, void), ID(3, 6, bar, REFERENCES(testFiles/parser/structDeclStmtOneField.tc, 3, 1)), STAB(), COMPOUNDSTMT(3, 12, STAB(ENTRY(s, STRUCT(testFiles/parser/structDeclStmtOneField.tc, 4, 3, FIELD(int, x)))), STRUCTDECL(4, 3, ID(4, 10, s, REFERENCES(testFiles/parser/structDeclStmtOneField.tc, 4, 3)), VARDECL(5, 5, KEYWORDTYPE(5, 5, int), ID(5, 9, x, REFERENCES()))))))
